### PR TITLE
Fix UG Bug: Make multiple field search feature clearer in UG

### LINKED ISSUE #199: [PE-D][Tester B] Find Command executes AND search instead of OR search
This image shows the original list of contacts

find n/irfan t/rejected

After executing the find command, only Irfan is found.
Both Roy and Charlotte have the "rejected" tag as well but is not listed after the command.

AND search is executed instead of OR search as stated in User Guide

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: MuhdMusab/ped#2

### PR DESCRIPTION
**Pull request by lulucopter**
Resolves #199

Make it clear in the UG for `find` command multiple field searches:

AND search across fields,
OR keyword search within fields

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -238,6 +238,7 @@ Format: `find parameter/KEYWORD [parameter/KEYWORD]...`
 * The search is case-insensitive. E.g., `bobby` will match `Bobby`.
 * Applicants matching at least one keyword will be returned (i.e., OR search). E.g., `n/Bobby Cortez` will return applicants with the name `Bobby Lacruz`, `Alexander Cortez`
```
Comment (maryjess) [inline docs/UserGuide.md:239]: Minor, but you might want to also consider italicising the `OR` here also

File: docs/UserGuide.md [hunk 2]
```diff
@@ -238,6 +238,7 @@ Format: `find parameter/KEYWORD [parameter/KEYWORD]...`
...
 * The order of the keywords does not matter. E.g., `Bobby cortez` will match `Cortez bobby`.
+* Multiple field search: Across different fields, an _AND_ search is done, while an _OR_ search is still done within each field for its keywords. E.g., `n\Bobby Cortez g/male` will return male applicants with names matching `Bobby` or `Cortez`.
```
Comment (maryjess) [inline docs/UserGuide.md:241]: Will it be better if we standardise the slash to be `/`?

Comment (lulucopter) [reply docs/UserGuide.md:241]: You are right. It was a typo on my end. Thanks for spotting it

**Review by cliftonfelix — APPROVED** [review]
LGTM

**Pull request merged**